assignment/README.md

```diff
@@ -37,7 +37,6 @@ Or alternatively
 
 Or to deploy only the bundle to the author, run
 
-    mvn clean install -PautoInstallBundle
```

Comment:
why is this file modified

Reply:
done by mistake , make be a space must be added to it when I was going through it sorry for that